solaris/src/convert.rs

```diff
@@ -35,3 +36,17 @@ pub fn bytes32(s: &str) -> [u8; 32] {
 pub fn uint(num: u64) -> types::U256 {
     num.into()
 }
+
+pub fn u256_from_bytes32(v: [u8; 32]) -> types::U256 {
```

Comment:
This doesn't match convention in this module.
Either: `uint_from_bytes32()` or just `u256` or `convert::uint_from_raw/convert::raw::to_uint(x)` (the reason for the last one being that, we use `convert:raw::uint(5)` to generate bytes)

Reply:
A bit too verbose imho: `uint_from_u64` or `uint_from_into_u256`?
I'm in favour of last proposal, where this method get's moved to `raw`. `raw` is implicitly `bytes`, so you have:
```
convert::raw::to_uint(convert:raw::uint(5)) == convert::uint(5)
```

Anyway, this module will be gone when we migrate primitives. @snd let me know if you want to work on introducing primitives to `ethabi`.